src/modules/claim/claim.types.ts

```diff
@@ -1,4 +1,10 @@
-export const NATS_EXCHANGE_TOPIC = 'claim.exchange';
+export const NATS_EXCHANGE_TOPIC = 'claim-exchange';
```

Comment:
What's the relationship between `NATS_EXCHANGE_TOPIC` and `NATS_SUBJECT_SUFFIX` from the `.env`?